app/resources/profile_link_resource.rb

```diff
@@ -0,0 +1,7 @@
+class ProfileLinkResource < BaseResource
+  immutable
```

Comment:
This probably shouldn't be immutable, that would mean nobody can edit the links on their profile